tests/tests.js

```diff
@@ -862,6 +882,96 @@ describe('Convert Lat/Lon to UTM', function(){
       chai.assert.equal(-2583444, parseInt(coords[1]));
       chai.assert.equal(38, coords[2]);
     });
+    describe('around Arizona in the United States', function(){
```

Comment:
✏️ I think all these new tests of the `withNS` function should include the N/S flag in the description of expected output.